nextbike/prediction/split.py

```diff
@@ -8,14 +8,14 @@
 
 
 def simple_split(df):
-    X = df.drop(["Place_start", "Place_end", "Duration"], axis=1)
```

Comment:
Warum droppst du den Place nicht mehr? Dachte das sei ein String?
Oder schon vorher woanders gedroppt?

Reply:
Not meaningful to drop features in split. this should be done before

Reply:
Done